src/modules/message/message.service.ts

```diff
@@ -48,19 +56,73 @@ export class MessageService {
     ) {
       await this.chatRoomService.changeStatus(data.chatRoomId);
     }
-
     const message = await this.messageRepository.save({
       text: data.text,
       chatRoom: { id: data.chatRoomId },
       user: { id: data.userId },
       messageType: MessageType.FROM_USER,
     });
 
-    await this.chatRoomService.updateAfterReceiveMessage(data.chatRoomId);
+    const receiverUserId =
+      chatRoom.freelancer.id === data.userId
+        ? chatRoom.client.id
+        : chatRoom.freelancer.id;
 
+    await this.chatRoomService.updateAfterReceiveMessage(data.chatRoomId);
+    await this.notificationsService.createMessageNotification(
+      message.id,
+      receiverUserId,
+    );
     return message;
   }
 
+  async createInitialMessages(
+    chatRoomId: number,
+    inviteFrom: number,
+    inviteTo: number,
+    type: ProposalType,
+    message: string,
+    bid: number,
+  ): Promise<void> {
+    await this.createSystemMessage({
+      text: `You have received a new ${
+        type === ProposalType.PROPOSAL ? 'proposal' : 'invite'
```

Comment:
maybe you should just let `You have received a new ${type}`.

Reply:
maybe you should just let `You have received a new ${type}`.

Reply:
I don't think so. Values of enum uses for functionality, and they can be renamed or be in camel cace etc, so I thinck it better use static text, it's just a coincidenc that they texts are same